server/src/projects/project-applicants/project-applicant.controller.ts

```diff
@@ -7,7 +7,11 @@ import { JsonApiDeserializePipe } from '../../json-api-deserialize.pipe';
 import { PROJECTAPPLICANT_ATTRS } from './project-applicants.attrs';
 import { CONTACT_ATTRS } from '../../contact/contacts.attrs';
 
-const ACTIVE_CODE = 1;
+const ACTIVE_STATUSCODE = 1;
+const INACTIVE_STATUSCODE = 2;
+
+const ACTIVE_STATECODE = 0;
+const INACTIVE_STATECODE = 1;
```

Comment:
just a warning, this might cause some conflicts -- I created some variables for these codes as well and named them slightly differently